core/renderers/block_rendering_rewrite/block_render_draw.js

```diff
@@ -203,15 +203,18 @@ Blockly.blockRendering.Drawer.prototype.drawStatementInput_ = function(row) {
   var x = input.xPos + input.width;
 
   var innerTopLeftCorner =
-      Blockly.blockRendering.constants.NOTCH.pathRight + ' h -' +
-      (Blockly.blockRendering.constants.NOTCH_WIDTH -
-          Blockly.blockRendering.constants.CORNER_RADIUS) +
+      input.notchShape.pathRight +
+      Blockly.utils.svgPaths.lineOnAxis('h',
+          -(Blockly.blockRendering.constants.NOTCH_OFFSET_LEFT -
```

Comment:
Can you get this notch_offset_left value to be stored on the measurable instead?

Reply:
Could also probably store a pointer to INSIDE_CORNERS, since I think that's used in the measurable already for sizing information.

Reply:
I like the change of storing notch_offset_left on the measurable. I was thinking of storing it on the notchShape object. Since with that change I should also update the other instances of notch_offset_left, I'd prefer to do that in a follow up PR.
As for inside_corners, I don't think it should be stored on a measurable since i don't anticipate different measurables having different corner shapes from each other.